tests/test.h

```diff
@@ -84,8 +84,8 @@ extern mtx_t test_mtx;
 #define TEST_LOCK()                     mtx_lock(&test_mtx)
 #define TEST_UNLOCK()                   mtx_unlock(&test_mtx)
 #define TEST_FIXTURES_FOLDER            "./fixtures"
-#define TEST_FIXTURES_KEYSTORE_PASSWORD "confluent"
-#define TEST_FIXTURES_KEY_PASSWORD      "confluent"
+#define TEST_FIXTURES_KEYSTORE_PASSWORD "strong_password"
```

Comment:
Is it possible to have different passwords for these, so that we also test the case that they are not mixed up internally in librdkafka?

Reply:
Also think these defines could reside in the test-file for now, until they're needed elsewhere.